src/base/bittorrent/session.cpp

```diff
@@ -321,6 +321,11 @@ QString Session::tempPath() const
     return m_tempPath;
 }
 
+QString Session::torrentTempPath(const InfoHash &hash) const
+{
+    return m_tempPath + QString(hash) + "/";
```

Comment:
I don't understand what you want to be saved?
>Neither the hash of the torrent shall appear in the filesystem, nor any derivative from it which leaves a possibility to restore the hash value or run a search with for it.

How can someone recover the hash, having only a part of it?